webpack.dev.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable */ 
```

Comment:
same here, check the error instead of disabling it.

Reply:
Thank you for your comments. :raised_hands: